keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/NamedImage.kt

```diff
@@ -0,0 +1,8 @@
+package com.netflix.spinnaker.keel.clouddriver.model
+
+class NamedImage(
+  val imageName: String,
+  val attributes: Map<String, Any?>,
+  val tagsByImageId: Map<String, Map<String, String?>?>,
+  val accounts: Set<String>,
+  val amis: Map<String, List<String>?>)
```

Comment:
region here?

Reply:
I'm not super happy with the typing and naming of these attributes but they are modelling the response from clouddriver's find endpoint. If this sticks around longer term then I'd like to make sure it is more clear that this is class is multi-regional